{"name": "Тест авторизации с неверным логином", "status": "passed", "start": 1709759542591, "stop": 1709759542754, "uuid": "eb6fdbf9-46a6-427f-bb5a-382d78ba0169", "historyId": "27943f6730ab4d740c33daa26147545a", "testCaseId": "27943f6730ab4d740c33daa26147545a", "fullName": "tests.test_login_user.TestLoginUser#test_login_user_incorrect_email_error", "labels": [{"name": "feature", "value": "Проверяем авторизацию пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "DESKTOP-AQ71LTK"}, {"name": "thread", "value": "24272-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}